templates/bundles/DemosPlanCoreBundle/DemosPlanAssessmentTable/DemosPlan/export_assessment_table_entry.tex.twig

```diff
@@ -58,7 +58,11 @@
     {% if statement.element.title is defined and statement.element.title != '' %} & {{ "document"|trans }}: & {{ statement.element.title|latex|raw }}{% if statement.document is defined and statement.document.title != '' %} / {{ statement.document.title|latex|raw }} {% endif %} \\ {% endif %}
     {% if statement.paragraph.title is defined and statement.paragraph.title != '' %} & {{ "paragraph"|trans }}:  & {{ statement.paragraph.title|latex|raw }}\\{% endif %}
     {% for attachment in statement.attachments %} & {{ "attachment.original"|trans }}: & {{ attachment.file.filename|latex|raw }}\\{% endfor %}
-    {% if statement.files|default([])|length > 0 %}{% for file in statement.files %} & {{ "file"|trans }}: & {{ file|getFile('name')|latex|raw }}\\{% endfor %}{% endif %}
+    {% if statement.files|default([])|length > 0 %}
+        {% for file in statement.files %}
+            & {{ "file"|trans }}: & {{ '{' }}{{ file|getFile('name')|replace({'_': '\_'})|latex|raw }}{{ '}' }}\\
```

Comment:
I dont think this change is necessary ```\usepackage{underscore}``` should replace _ with \_ automatically.

Reply:
You need to be extremely careful with spaces in LaTeX. This is why the templates are so malformed. You cannot just introduce newlines unless you exactly know what you do